RELEASE/scripts/sl_ascend.ash

```diff
@@ -1205,6 +1205,11 @@ boolean doThemtharHills()
 	}
 	int expectedMeat = numeric_modifier("Generated:_spec", "meat drop");
 
+
+	if(get_property("sl_useWishes").to_boolean())
+	{
+		makeGenieWish($effect[Frosty]);
```

Comment:
This should also be wrapped in a check for if you already have Frosty, so you don't burn through all your wishes wishing for Frosty as turns go by.